lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -31,7 +31,16 @@ module.exports = {
 
   createStack() {
     const stackName = `${this.serverless.service.service}-${this.options.stage}`;
-
+    if (/^[^a-zA-Z].+|.*[^a-zA-Z0-9\-].*/.test(stackName) || stackName.length > 128) {
+      const errorMessage = [
+        `The stack name "${stackName}" is not quallify. `,
+        'A stack name can contain only alphanumeric',
+        ' (case sensitive) and hyphens. It must characters',
```

Comment:
Could you update it like `It should start with an alphabetic character and shouldn't exceed 128 characters.`

Thank you 👍